server/src/main/java/org/elasticsearch/action/admin/indices/alias/Alias.java

```diff
@@ -248,6 +249,11 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         return builder;
     }
 
+    @Override
+    public String toString() {
```

Comment:
I added this because during debugging #30911 I found it really useful to have a readable toString() output for the Alias class.

Reply:
+1